cylc/flow/cfgspec/globalcfg.py

```diff
@@ -99,6 +99,53 @@
         'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
     },
 
+    # job platforms
+    'job platforms': {
+        '__MANY__': {
+            'batch system': [VDR.V_STRING, 'background'],
+            'batch submit command template': [VDR.V_STRING],
+            'shell': [VDR.V_STRING, '/bin/bash'],
+            'run directory': [VDR.V_STRING, '$HOME/cylc-run'],
+            'work directory': [VDR.V_STRING, '$HOME/cylc-run'],
+            'suite definition directory': [VDR.V_STRING],
+            'task communication method': [
+                VDR.V_STRING, 'zmq', 'poll'],
+            # TODO ensure that it is possible to over-ride the following three
+            # settings in suite config.
+            'submission polling intervals': [VDR.V_INTERVAL_LIST],
+            'submission retry delays': [VDR.V_INTERVAL_LIST, None],
```

Comment:
This one is new (not currently configurable for hosts) but I think it makes sense.